ts/features/bonus/bpd/api/backendBpdClient.ts

```diff
@@ -297,9 +342,13 @@ export function BackendBpdClient(
 
   return {
     find: withBearerToken(createFetchRequestForApi(findT, options)),
```

Comment:
Could we also sign as deprecated here?
```suggestion
     /**
     * @deprecated
     */
    find: withBearerToken(createFetchRequestForApi(findT, options)),
```

Reply:
Yes, setted as deprecated in [5f8e4e1](https://github.com/pagopa/io-app/pull/2969/commits/5f8e4e1f68e5488790b132f561a048273aabe30f)